src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -520,8 +520,8 @@ describe('<ReviewCollapsibleChapter>', () => {
     );
 
     const titleDiv = wrapper.find('.form-review-panel-page-header');
-    expect(titleDiv.length).to.equal(1);
-    expect(titleDiv.text()).to.equal('');
+    // Title is no longer rendered if it contains an empty string
```

Comment:
I would reserve "no longer" comments like these for the PR to indicate changes. If it's important to point out that the thing shouldn't be rendered, you can keep the comment without the "no longer", as in "Title is not rendered if it contains an empty string."

Reply:
Oh wow! I'm sorry!! 😞 

I start my vacation early tomorrow morning, so I rushed to get this merged and didn't see all your comments. I'll make a note to look at it again when I return (if not already done).